### PR TITLE
add spec for user behaviour

### PR DESCRIPTION
**Pull request by chimedpalden**

### CODE REVIEW

File: spec/harvesting/models/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  describe "#fetch_next_page" do
+    it "provides next page content" do
+      stub_request(:get, /users/).
+            with(query: {"page" => "2"}).
+            to_return( { body: page2_results.to_json } )
+
+      expect(users.entries.count).to eq(100)
+      expect(users.page).to eq(1)
+      users.send(:fetch_next_page)
```
Comment (etagwerker) [inline spec/harvesting/models/users_spec.rb]: @chimedpalden I think it's best to not know so much about the implementation of the pagination logic... 

Could you change this scenario to iterate through `user.entries` 115 times? That should do what you expect it to do.

**Comment by bronzdoc** [comment]
I'll pick this up

**Comment by bronzdoc** [comment]
@cleicar are you going to work on this?

**Comment by etagwerker** [comment]
@bronzdoc @chimedpalden I just reviewed this again and it seems that https://github.com/ombulabs/harvesting/pull/17 added coverage for this behavior. 

Thanks!

**Pull request closed without merging**